src/current/_includes/v23.1/ui/insights.md

```diff
@@ -127,30 +106,100 @@ To view [details of the execution](#statement-execution-details), click an execu
 
-{% if page.cloud != true -%}
-The following screenshot shows the execution details of the statement execution in the preceding section:
-
-<img src="{{ 'images/v22.2/statement_execution_details.png' | relative_url }}" alt="Statement execution details" style="border:1px solid #eee;max-width:100%" />
-{% endif -%}
-
-The **Insights** column shows the name of the insight, in this case **Suboptimal Plan**; the **Details** column provides details on the insight; and the final column contains a **Create Index** button. Click the **Create Index** button to perform a query to mitigate the cause of the insight, in this case to create an index on the ride `start_time` that stores the `rider_id`.
-
 - **Start Time**: The timestamp when the statement execution started.
 - **End Time**: The timestamp when the statement execution ended.
 - **Elapsed Time**: The time that elapsed during statement execution.
-- **Rows Read**: The total number of rows read.
-- **Rows Written**: The total number of rows written.
+- **CPU Time**: The amount of CPU time spent executing the statement. The CPU time represents the time spent and work done within SQL execution operators.
+- **Rows Read**: The total number of rows read by the statement execution.
+- **Rows Written**: The total number of rows written by the statement execution.
 - **Transaction Priority**: The [priority]({{ link_prefix }}transactions.html#set-transaction-priority) of the transaction for the statement execution.
 - **Full Scan**: Whether the execution performed a full scan of the table.
+- **Transaction Retries**: The total number of retries of the transaction for the statement execution.
 - **Session ID**: The ID of the [session]({{ link_prefix }}ui-sessions-page.html) the statement was executed from.
 - **Transaction Fingerprint ID**: The ID of the transaction fingerprint for the statement execution.
 - **Transaction Execution ID**: The ID of the transaction execution for the statement execution.
 - **Statement Fingerprint ID**: The fingerprint ID of the statement fingerprint for the statement execution.
 
+#### Statement with ID {transaction ID} waited on
+
+This section provides details of the transaction executions that block the statement ID flagged with [**High Contention**](#high-contention):
+
+- **Transaction Execution ID**: The execution ID of the blocking transaction execution.
+- **Transaction Fingerprint ID**: The transaction fingerprint ID of the blocking transaction execution.
+- **Contention Time**: The time that transactions with this execution ID were [in contention]({{ link_prefix }}performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention) with other transactions within the specified time interval.
+- **Database Name**: The name of the contended database.
+- **Schema Name**: The name of the contended schema.
+- **Table Name**: The name of the contended table.
+- **Index Name**: The name of the contended index.
+
+## Workload Insight Types
```

Comment:
style nit: Headings within a page should be sentence case, so "Types" should be "types"

Also, I would suggest adding a sentence in this section that introduces what the rest of the subheadings here will cover. Something like, 

```
The Workload Insights tab surfaces the following types of insights:
- [Failed Execution](#link to heading)
- [High Contention](#link)
- ... etc.
```

Reply:
Done